content/pages/what-is-mongock.md

```diff
@@ -7,7 +7,49 @@ eleventyNavigation:
   title: What is Mongock?
   order: 0
 ---
-Mongock is a java MongoDB tool for tracking, managing and applying database schema changes across all your environments based on a coding approach.
-**NEEDS TO BE CHANGED**
+Mongock is a Java based Migration tool as part of your Application code. It allows Developers to have ownership and control over Data migrations during the deployment process, allowing safer migrations in Distributed Systems as code and data changes are shipped together. 
 
+It was originally designed for MongoDB Data migrations and the product is evolving to provide a wider suite of NoSQL / SQL Database compatibility and cover other feautures for transactional execution use cases that require state management in Distributed systems. 
 
+**Why Mongock?**
+Our mission is to enable Developers to deploy and manage stateful Data migrations in Distributed Systems in a safe and reliable manner. If you are looking for migrating data for your Java Application Services in a safe and reliable manner via Code, Mongock is your best bet.
+
+There are several features Mongock enables you in your project:
+- Mongock is a Java-based tool that you can import in your Application.
+- Mongock promotes a Code-first approach for Migrations, allowing you to write your migration scripts in Java wh will ship with your Application code.
+- Mongock persists in your DB the changes made, allowing you to gain more control over the management of DB changes via Migrations, Rollbacks + other amazing features.
+- It is the most reliable production-grade solution for MongoDB migrations currently in the market, compatible with Mongo Atlas and different MongoDB versions.
```

Comment:
Do we want to specify MongoDB here, or reserve it to the MongoDB driver?

Reply:
I put the MongoDB one in the main page because of 2 main reasons:
1. It is the only reliable migration tool for MongoDB - which is the main market differentiator with competitors.
2. It's origins was MongoDB
I get your point around the driver, but that feels more a technical implementation concern rather than Why to use our product? 
Happy to get thoughts as I had the same dilemma when I wrote it down and chose to include it :)

Reply:
I buy it, we can add other "arguments" related to other database as we add drivers